### DISCUSSION TITLE
actions/upload-artifact failing unexpectedly with Error Code 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by mattzech** [community/community#41848] [Actions]
### Select Topic Area

Bug

### Body

I am curious to see if this is happening to anyone else. I have tried the following:
- Using a unique artifact name
- Updating to the most recent github runner version v2.300.0
- Using both v2 and v3 of upload-artifact

**Comment by quzanh1130** [comment]
idk man. i cant help this one :)

**Reply by mattzech** [reply]
It's really weird, very vague error message
